care/utils/tests/test_base.py

```diff
@@ -193,10 +193,12 @@ def setUpClass(cls) -> None:
         cls.super_user = cls.create_super_user(district=cls.district)
         cls.facility = cls.create_facility(cls.district)
         cls.patient = cls.create_patient()
+        # cls.ambulance = cls.create_ambulance()
```

Comment:
This is because I have started writing tests for ambulance API but it is a work in progress